src/rust/engine/src/nodes.rs

```diff
@@ -923,33 +897,17 @@ pub struct Snapshot {
 
 impl Snapshot {
   fn create(context: Context, path_globs: PathGlobs) -> NodeFuture<fs::Snapshot> {
-    // Recursively expand PathGlobs into PathStats while tracking their dependencies.
+    // Recursively expand PathGlobs into PathStats.
+    // We rely on Context::expand tracking dependencies for scandirs,
```

Comment:
(technically it's tracking both `ReadLink`s and `Scandir`s)

Reply:
Done